docs/source/conf.py

```diff
@@ -0,0 +1,39 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+
+import sphinx_rtd_theme
+
+project = 'casual-java'
+copyright = '2024, casualproject'
+author = 'casualproject'
+release = '3.2.33'
```

Comment:
Would it be possible to derive the docs version from the casual-java version? Will they always  be the exact same?

Reply:
Yeah, we can get the current version from versions.gradle